app/uk/gov/hmrc/trackingconsentfrontend/assets/src/settingsFormHandler.ts

```diff
@@ -0,0 +1,31 @@
+import castToArray from './castToArray'
+
+const settingsFormHandler = (document, userPreferences) => {
+  document.addEventListener('DOMContentLoaded', () => {
```

Comment:
We should listen for this in the entrypoint module for consistency with the other bundle. Or move both.

Reply:
What's the test strategy for that?